src/logger/debugContext.ts

```diff
@@ -7,4 +7,6 @@
  */
 export const DebugContext = {
   // e.g. composer: 'composer'
+  session: 'session',
+  notifications: 'notifications',
```

Comment:
Added these two. Thinking we actually may want to be able to silence certain debugs also (inverse of what is possible rn). Can layer that on later.